tests/integ/test_cte.py

```diff
@@ -100,6 +104,12 @@ def count_number_of_ctes(query):
         lambda x: x.to_df("a1", "b1").alias("L"),
     ],
 )
+@sql_count_checker(
```

Comment:
how about also adding describe_count here and other tests?

Reply:
the describe count check is embeded in check_result, which is used in this test point already.
there is also a limitation that the sql_counter_check annotation currently seems not working with describe_count yet, a ticket has been created here to https://snowflakecomputing.atlassian.net/browse/SNOW-1704919 to track the issue